script/sync-ghes/index.ts

```diff
@@ -59,6 +61,7 @@ async function checkWorkflows(
 
         const enabled =
           !isPartnerWorkflow &&
+          workflowProperties.enterprise !== false &&
```

Comment:
```suggestion
          workflowProperties.enterprise &&
```

Do we need the explicit `!== false`?

Reply:
I think we do, yes. The field can also be `undefined` which I think we want to treat the same as it being true.